handlers/authHandlers.js

```diff
@@ -27,49 +28,103 @@ if (!fs.existsSync('./keys/key') && !fs.existsSync('./keys/key.pub') && GENERATE
   }
 }
 
-try {
-  const prikeyPath = './keys/key';
-  if (fs.existsSync(prikeyPath)) {
-    PRIKEY = fs.readFileSync(prikeyPath, 'utf8');
-  } else {
-    if (DISABLE_SEC) {
-      PRIKEY = '';
-      console.warn('prikey null since DISABLE_SEC and no prikey provided');
+if (ENABLE_SECURITY_AT) {
```

Comment:
I imagine this could be done with less complexity/duplication by setting an intermediate variable in place of how disable security is used now. That way we won't need two different ways to read our pub/pri keys.

Reply:
Okay, I'll figure this out and get back to you. Thanks.

Reply:
@birm Thanks, I have done the changes and the PRs are up for further review.